routes/login.js

```diff
@@ -30,4 +30,120 @@ router.get('/login', (req, res) => {
   return null
 })
 
+router.post('/register', (req, res, next) => {
+
+	pg.connect(connectionString, (err, client, done) => {
+
+		// Handle connection errors
+		if(err) {
+			done();
+			console.log(err);
+			return res.status(500).json({success: false, data: err});
+		}
+
+		//Add a Staff
+		let queryText = `
+		INSERT 
+		INTO HMS-PMS.Staff
+		VALUES (
+		'${req.body.staffId}',
```

Comment:
Should always be userID, it changes between staffId and userId

Reply:
we will change it to staffID everywhere. do the same please

Reply:
Okay